django-stubs/core/files/base.pyi

```diff
@@ -8,7 +8,7 @@ _T = TypeVar("_T", bound="File")
 
 class File(FileProxyMixin, IO[Any]):
     DEFAULT_CHUNK_SIZE: Any = ...
-    file: StringIO = ...
+    file: IO[Any] = ...
```

Comment:
Is it possible to replace `Any` with something more specific here? Or is it the only way?

Reply:
I think it might be the only way. The opened file being assigned to the attribute may be either opened as bytes or a string.

Also, from a brief look at how `open()` works in mypy, if the `mode` arg is constant it can infer if it is either `IO[str]` or `IO[bytes]` but if it is a variable it will return `IO[Any]`. Django's `File.open()` method uses a variable for the `mode` arg which means the type in django will be `IO[Any]`.

Reply:
Thanks!